pkg/libvmi/storage.go

```diff
@@ -118,6 +126,13 @@ func WithHostDisk(diskName, path string, diskType v1.HostDiskType) Option {
 	}
 }
 
+func WithHostDiskAndCapacity(diskName, path string, diskType v1.HostDiskType, capacity string) Option {
```

Comment:
Same, make `WithHostDisk` call `WithHostDiskAndCapacity`? And pass `defaultDiskSize` to `WithHostDiskAndCapacity`.